src/atomate2/lobster/schemas.py

```diff
@@ -0,0 +1,585 @@
+            are_coops=True,
+            relevant_bonds=analyse.final_dict_bonds,
+        )
+        icobi_bond_dict = get_strng_bonds(
+            icobi_dict,
+            are_cobis=True,
+            are_coops=False,
+            relevant_bonds=analyse.final_dict_bonds,
+        )
+
+        sb_ichop = StrongestBonds(
+            are_cobis=False,
+            are_coops=False,
+            strongest_bonds=icohp_bond_dict,
+            which_bonds=analyse.whichbonds,
+        )
+        sb_icoop = StrongestBonds(
+            are_cobis=False,
+            are_coops=True,
+            strongest_bonds=icoop_bond_dict,
+            which_bonds=analyse.whichbonds,
+        )
+        sb_icobi = StrongestBonds(
+            are_cobis=True,
+            are_coops=False,
+            strongest_bonds=icobi_bond_dict,
+            which_bonds=analyse.whichbonds,
+        )
+
+        cohp_obj = CompleteCohp.from_file(
+            fmt="LOBSTER",
+            structure_file="POSCAR.gz",
+            filename="COHPCAR.lobster.gz",
+            are_coops=False,
+            are_cobis=False,
+        )
+
+        coop_obj = CompleteCohp.from_file(
+            fmt="LOBSTER",
+            structure_file="POSCAR.gz",
+            filename="COOPCAR.lobster.gz",
+            are_coops=True,
+            are_cobis=False,
+        )
+
+        cobi_obj = CompleteCohp.from_file(
+            fmt="LOBSTER",
+            structure_file="POSCAR.gz",
+            filename="COBICAR.lobster.gz",
+            are_coops=False,
+            are_cobis=True,
+        )
+        doscar_lobster = Doscar(doscar="DOSCAR.lobster.gz", structure_file="POSCAR.gz")
+        dos = doscar_lobster.completedos
+        lso_dos = None
+        if additional_fields and "DOSCAR.LSO.lobster" in additional_fields:
+            doscar_lso_lobster = Doscar(
+                doscar="DOSCAR.LSO.lobster.gz", structure_file="POSCAR.gz"
+            )
+            lso_dos = doscar_lso_lobster.completedos
+
+        madelung_obj = MadelungEnergies(filename="MadelungEnergies.lobster.gz")
+
+        madelung_energies = {
+            "Mulliken": madelung_obj.madelungenergies_Mulliken,
+            "Loewdin": madelung_obj.madelungenergies_Loewdin,
+            "Ewald_splitting": madelung_obj.ewald_splitting,
+        }
+
+        return cls(
+            structure=struct,
+            dir_name=dir_name,
+            lobsterin=lobster_in,
+            lobsterout=lobster_out,
+            lobsterpy_data=lpca,
+            lobsterpy_summary_text=" ".join(describe.text),
+            strongest_bonds_icohp=sb_ichop,
+            strongest_bonds_icoop=sb_icoop,
+            strongest_bonds_icobi=sb_icobi,
+            cohp_data=cohp_obj,
+            coop_data=coop_obj,
+            cobi_data=cobi_obj,
+            dos=dos,
+            lso_dos=lso_dos,
+            charges=charges,
+            madelung_energies=madelung_energies,
+        )  # doc
+
+
+def _get_structure() -> Structure:
```

Comment:
Is this needed or just copy and paste from amset schemas file?

Reply:
Yes, it was a copy and paste error. Sorry!